classes/class-p4-master-site.php

```diff
@@ -783,4 +784,29 @@ public function gdpr_cc_comment_form_add_class( $submit_field, $args ) {
 
 		return $submit_field;
 	}
+
+	/**
+	 * Filter function for embed_oembed_html.
+	 * Transform youtube embeds to youtube-nocookie.
+	 *
+	 * @see https://developer.wordpress.org/reference/hooks/embed_oembed_html/
+	 *
+	 * @param mixed  $cache The cached HTML result, stored in post meta.
+	 * @param string $url   The attempted embed URL.
+	 *
+	 * @return mixed
+	 */
+	public function filter_youtube_oembed_nocookie( $cache, $url ) {
+		if ( ! empty( $url ) ) {
+			if ( strpos( $url, 'youtube.com' ) !== false ) {
+				$cache = str_replace( 'youtube.com', 'youtube-nocookie.com', $cache );
+			}
+			if ( strpos( $url, 'youtu.be' ) !== false ) {
+				$cache = str_replace( 'youtube.com', 'youtube-nocookie.com', $cache );
```

Comment:
should replace `youtu.be` part here?

Reply:
nope. at this point the html from the embed is filtered, which contains the `youtube.com` domain although the original embed url is `youtu.be`

Reply:
can we merge it then into one if? since we are doing the same thing in both if/else